src/yvalve/why.cpp

```diff
@@ -6515,83 +6553,122 @@ YService* Dispatcher::attachServiceManager(CheckStatusWrapper* status, const cha
 			IntlSpb().toUtf8(spbWriter);
 		}
 
-		// Build correct config
-		RefPtr<const Config> config(Config::getDefaultConfig());
-		if (spbWriter.find(isc_spb_config))
+		IProvider* p;
+		service = internalServiceAttach(status, svcName, spbWriter,
+			[](CheckStatusWrapper*, IService*){ }, Dispatcher::PTYPE_ANY, &p);
```

Comment:
Can't this empty lambda be just `{}`?

Reply:
Can - but I see no use in it, in that case additional null check in internalServiceAttach() needed.